packages/remix-forms/src/createForm.tsx

```diff
@@ -302,7 +302,20 @@ function createForm({
       } as Field<SchemaType>
     }
 
-    const globalErrors = errors?._global
+    const deepHiddenFieldsErrors = hiddenFields?.map((hiddenField) => {
+      const hiddenFieldErrors = fieldErrors(hiddenField)
+
+      if (hiddenFieldErrors?.length) {
+        const hiddenFieldLabel =
+          (labels && labels[hiddenField]) || inferLabel(String(hiddenField))
+        return hiddenFieldErrors.map((error) => `${hiddenFieldLabel}: ${error}`)
+      } else return []
+    })
+    const hiddenFieldsErrors: string[] = deepHiddenFieldsErrors?.flat() || []
+
+    let globalErrors = ([] as string[])
+      .concat(errors?._global || [], hiddenFieldsErrors)
+      .filter((error) => typeof error === 'string')
```

Comment:
Some types shenanigans, I accept other ideas

Reply:
what are the other cases for error?

Reply:
`undefined`